core/src/main/scala/chisel3/Bits.scala

```diff
@@ -94,9 +100,9 @@ sealed abstract class Bits(private[chisel3] val width: Width) extends Element wi
     * @param x an index
     * @return the specified bit
     */
-  final def apply(x: BigInt): Bool = macro SourceInfoTransform.xArg
+  final def apply(x: BigInt): Bool = macro IntLiteralApplyTransform.safeApply
 
-  /** @group SourceInfoTransformMacro */
+  /** @group IntLiteralApplyTransformMacro */
```

Comment:
```suggestion
  /** @group SourceInfoTransformMacro */
```
This group is not related to the actual `macro` class, it's purely used for ScalaDoc grouping, leave that bit alone.